File: 09-01-2025_Counting_Words_With_a_Given_Prefix.py
class Solution:
    def prefixCount(self, words, pref):
        count = 0
        len_pref = len(pref)
        for word in words:
            if len_pref <= len(word):
                sub = word[:len_pref]
                if pref == sub:
                    count += 1
        return count

File: 09-01-2025_Counting_Words_With_a_Given_Prefix.cpp
#include <string>
#include <vector>
using namespace std;

class Solution {
public:
    int prefixCount(vector<string>& words, string pref) {
        int count = 0;
        int len = pref.length();
        for (const string& word : words) {
            if (len <= word.length()) {
                string sub = word.substr(0, len);
                if (pref == sub) {
                    count++;
                }
            }
        }
        return count;
    }
};